.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,88 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI React Native? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+<!-- Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Community. -->
+
+### Component ownership
+
+<!-- Are there one or more people who can help maintain this component over time? Who will address bugs? -->
+
+## Component composition
+
+### Imports/Dependencies
+
+<!-- What dependencies will your component be using -->
+
+### Proposed Public API
+
+<!-- Define the proposed public API for this control -->
+
+### Intended Platforms
+- [ ] iOS
+- [ ] macOS
+- [ ] win32
+- [ ] windows
+- [ ] android
+
+## Component code
+
+### Example code
+
+```javascript
+```
+
+### New Component Checklist
+
+General:
+- [ ] Automated Tests
+- [ ] Documentation and examples
+- [ ] Keyboard Accessibility
+- [ ] Internationalization and Right-to-left Layouts
+- [ ] Design Review
+- [ ] API review
```

Comment:
What do these mean? Until we have a formal process to link to maybe we don't add it? Or do we have one?

Reply:
We don't have a formal process I know about. I can just take it out until we have one